### ISSUE TITLE
Tachyons via cdn?

### ISSUE THREAD

**Issue by jchck** [tinychime/jekyons#14]
hey, thanks @joshosbrn for making this and sharing with the whole wide world!! 

it's pretty super great.

just curious, why'd you opt to include Tachyons via cdn and not npm?

thanks!

**Comment by joshosbrn** [comment]
Hi there, @jchck! The original decision to go CDN instead of NPM was to save time, and honestly I wasn't as familiar with NPM when I built Jekyons. I believe @alexcarpenter made the Tachyons version number a variable in a PR a few months ago and that's felt right so far.

All of that to say, I'm definitely not opposed to moving to NPM. IMO the big drawback (obviously, I think) to using CDN for this is the inability to work offline. It looks like you have some commits on your fork; I'd love to check out a PR & discuss there as a group whenever you're ready!

Thanks! 👒

**Comment by jchck** [comment]
Ah, makes sense. 

Cool, I'll clean up what I have a submit PR.

**Comment by alexcarpenter** [comment]
@jchck @joshosbrn Yeah, from what I understand, Jekyons is a prototyping tool (not to say you cannot use if for production sites). Being able to get up and running quickly meant relying on the CDN version. Easy to maintain and no compiling needed. Just my two cents.
